bot/__main__.py

```diff
@@ -1,10 +1,16 @@
 import discord
+import sentry_sdk
 from discord.ext.commands import when_mentioned_or
+from sentry_sdk.integrations.aiohttp import AioHttpIntegration
 
 from bot import patches
 from bot.bot import Bot
 from bot.constants import Bot as BotConfig, DEBUG_MODE
 
+sentry_sdk.init(
+    dsn=BotConfig.sentry_dsn,
+    integrations=[AioHttpIntegration()]
```

Comment:
Is this going to complain about not being created inside an async function?

Reply:
It won't but I have just realised this is intended for aiohttp servers, not clients.

https://docs.sentry.io/platforms/python/aiohttp/